RUFAS/routines/field/soil/manure_pool.py

```diff
@@ -67,4 +82,685 @@ def __eq__(self, other: Union["ManurePool", object]) -> Any:
             and self.stable_organic_phosphorus == other.stable_organic_phosphorus
             and self.organic_phosphorus_runoff == other.organic_phosphorus_runoff
             and self.inorganic_phosphorus_runoff == other.inorganic_phosphorus_runoff
+            and self.annual_runoff_manure_organic_phosphorus == other.annual_runoff_manure_organic_phosphorus
+            and self.annual_runoff_manure_inorganic_phosphorus == other.annual_runoff_manure_inorganic_phosphorus
+        )
+
+    def daily_manure_update(
+        self,
+        rainfall: float,
+        field_size: float,
+        mean_air_temperature: float,
+    ) -> float:
+        """
+        This method conducts daily operations on manure including decomposition, assimilation and returns the total
+        assimilation.
+
+        Parameters
+        ----------
+        rainfall : float
+            The amount of rainfall on the current day (mm).
+        field_size : float
+            The size of the field (ha).
+        mean_air_temperature : float
+            Mean air temperature on the current day (degrees C).
+
+        return
+        ------
+        float
+            The total amount of assimilated phosphorus (kg).
+        """
+        temperature_factor = self._determine_temperature_factor(mean_air_temperature)
+        if rainfall < 1 or rainfall > 4:
+            self.adjust_manure_moisture_factor(rainfall, temperature_factor)
+
+        # Calculate manure decomposition on soil surface
+        decomposed_mass, decomposed_coverage = self.determine_decomposed_surface_manure(temperature_factor)
+
+        # Calculate phosphorus mineralization between pools
+        mineralized_stable_organic = self.determine_mineralized_surface_phosphorus(
```

Comment:
Are these comments necessary?
```suggestion
        decomposed_mass, decomposed_coverage = self.determine_decomposed_surface_manure(temperature_factor)

        mineralized_stable_organic = self.determine_mineralized_surface_phosphorus(
```

Reply:
They're originally there, I think they're somewhat helpful. Let me know what you think I'm also fine with removing them.

Reply:
Looking at it again i will remove those.